ui/app/AppLayouts/Wallet/WalletUtils.qml

```diff
@@ -42,6 +42,9 @@ QtObject {
       rationale: https://github.com/status-im/status-desktop/pull/14959#discussion_r1627110880
       */
     function calculateMaxSafeSendAmount(value, symbol) {
+        if (!value) {
```

Comment:
should line no 48 handle this already?

Reply:
value can be undefined as well. In this case I'm just returning 0 as max send amount.